scripts/ort-evaluator.sh

```diff
@@ -4,13 +4,16 @@
 # against customizable policy rules.
 
 if [[ "$ORT_DISABLE_SCANNER" = "true" ]]; then
-    ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ANALYZER_FILE
-elif [[ "$ORT_DISABLE_ADVISOR" = "false" ]]; then
-    ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ADVISOR_FILE
+    if [[ "$ORT_DISABLE_ADVISOR" = "false" ]]; then
+        ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ADVISOR_FILE
+    else
+        ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ANALYZER_FILE
+    fi
 else
     ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_SCANNER_FILE
 fi
 
+
```

Comment:
Nit: Unrelated change.

Reply:
This unrelated line change is still present in the diff.

Reply:
should be fine now